ambari-metrics-common/pom.xml

```diff
@@ -103,7 +103,6 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>28.0-jre</version>
```

Comment:
it would be good to provide a version here ?

Reply:
I think the parent pom's guava version should be used, so I deleted the separate version in here